examples/calculator/src/calculator_android.nit

```diff
@@ -22,16 +22,18 @@ module calculator_android is
 
 	# Use a translucent background and lock in portrait mode
 	android_manifest_activity """
-		android:theme="@android:style/Theme.Holo.Wallpaper"
 		android:screenOrientation="portrait""""
 end
 
-import android
+# FIXME the next line should import `android` only when it uses nit_activity
+import android::log
```

Comment:
It should be `import android` but for now it also imports `native_app_glue`. So when `android` uses `nit_activity` instead of `native_app_glue`, when can change the import.